sdk/monitor/monitor-ingestion/samples-dev/concurrency.ts

```diff
@@ -29,12 +29,19 @@ async function main() {
 
   // The logs will be split into multiple batches and uploaded concurrently. By default,
   // the maximum number of concurrent uploads is 5.
-  const result = await client.upload(ruleId, streamName, logs);
-  if (result.status !== "Success") {
-    console.log("Some logs have failed to complete ingestion. Upload status=", result.status);
-    for (const errors of result.errors) {
-      console.log(`Error - ${JSON.stringify(errors.cause)}`);
-      console.log(`Log - ${JSON.stringify(errors.failedLogs)}`);
+  try {
+    await client.upload(ruleId, streamName, logs);
```

Comment:
What's the difference for our users between this and providing a `onError` instead of try/catch?

Reply:
It may be good to mention if uploads will try to send all logs or does it stop trying on the first failure

Reply:
It sends all logs unless it is aborted explicitly